internal/app/connectconformance/connectconformance.go

```diff
@@ -356,6 +371,15 @@ func run( //nolint:gocyclo
 						return err
 					}
 
+					// Double-check that client is still running before spawning a server process.
```

Comment:
Oops, we were previously printing the "Running X tests with ..." message, even if we didn't actually run anything because the client stopped. A simple matter of putting that verbose logging _after_ this check, instead of doing it before.